pull_request_template.md

```diff
@@ -1,16 +1,21 @@
-## Purpose
-_Describe the problem or feature in addition to a link to the issues._
+## Type
+- [ ] Feature
+- [ ] Bug
+- [ ] Update (fix something that did not work as expected)
+- [ ] Documentation
 
-## Approach
-_How does this change address the problem?_
+## Purpose
+_Why does this exist?_
 
-#### Open Questions and Pre-Merge TODOs
-- [ ] Use github checklists. When solved, check the box and explain the answer.
+## Solution
+_What changed, and does this address the problem? _
 
-## Learning
-_Describe the research stage_
+## Tests
+_Was this tested? How?_
 
-_Links to blog posts, patterns, libraries or addons used to solve this problem_
+#### Checklist
+- [ ] Ran tests successfully
+- [ ] Errors handled or acknowledged
```

Comment:
I think we can replace these two with just "Added or updated unit tests", since we'll want to make sure to do that for every change. The CI will make sure that all the tests are passing.

Reply:
Agreed. The checklist should be for things that are sometimes forgotten and can't easily be checked in CI, and that includes adding/updating tests for all new/changed behaviour. CI will tell us that the tests passed, and error handling is more of a design/implementation thing that doesn't apply to all PRs.